src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
+// #region body
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+// #endregion
+.card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  align-items: start;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    color: var(--Main-accent, #060b35);
```

Comment:
If you use `.scss`, you can also write variables in the style of this technology and keep them separate from the main rules.